sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -54,6 +54,9 @@ object ScalaNativePlugin extends AutoPlugin {
     val nativeDump =
       taskKey[Boolean](
         "Shall native toolchain dump intermediate NIR to disk during linking?")
+
+    val targetTriple =
+      taskKey[String]("The optional LLVM target triple for cross compiling.")
```

Comment:
We shouldn't add this in the new public API. The new way to configure things is to transform `nativeConfig`, for all settings. All the other keys, like `nativeGC`, are there for compatibility reasons, but we shouldn't add *more* of them. Users should use the following config instead:
```scala
nativeConfig ~= { _.withTargetTriple("whatever") }
```

Reply:
Thank you for the review, I understand now. I will clean this up and add some documentation as needed.